frontend/src/app/app.tsx

```diff
@@ -1,15 +1,44 @@
+import { actions as authActions } from '~/bundles/auth/store/auth.js';
 import {
+    Loader,
     Notifications,
     PageLayout,
     RouterOutlet,
 } from '~/bundles/common/components/components.js';
+import {
+    useAppDispatch,
+    useAppSelector,
+    useEffect,
+    useState,
+} from '~/bundles/common/hooks/hooks.js';
 
 const App: React.FC = () => {
+    const dispatch = useAppDispatch();
+    const { dataStatus } = useAppSelector(({ auth }) => ({
+        dataStatus: auth.dataStatus,
+    }));
+
+    const [isUserFullfilled, setIsUserFullfilled] = useState(false);
+
+    useEffect(() => {
+        setIsUserFullfilled(
+            dataStatus === 'fulfilled' || dataStatus == 'rejected',
+        );
+    }, [dataStatus]);
+
```

Comment:
You don't need this code, unnecessary complication
```suggestion
    const isLoading = dataStatus === DataStatus.PENDING; 

```

Reply:
Could I leave this code? Dasha is working on it. I will merge updates after she finishes that part.